web/src/components/tables/sample-table.tsx

```diff
@@ -15,10 +15,12 @@ type Props = {
   height?: number;
   minRows?: number;
   stretchH?: 'none' | 'all' | 'last';
+  setFilteredSamples: (samples: string[]) => void;
+  sampleTableIndex: string;
```

Comment:
So, this `<SampleTable />` component is used in a few places in the app... so should we set this as optional using the `?` operator in the Props definition?